### ISSUE TITLE
Maximum size of isolate log file?

### ISSUE THREAD

**Issue by pbadenski** [thlorenz/deoptigate#18]
Is there a maximum supported file size for isolate log file?

Getting this for an 800MB isolate-*-v8.log file:
```
node:buffer:592
    slice: (buf, start, end) => buf.utf8Slice(start, end),
                                    ^

Error: Cannot create a string longer than 0x1fffffe8 characters
    at Object.slice (node:buffer:592:37)
    at Buffer.toString (node:buffer:789:14)
    at Object.readFileSync (node:fs:421:41)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1110:18)
    at Module.load (node:internal/modules/cjs/loader:948:32)
    at Function.Module._load (node:internal/modules/cjs/loader:789:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:72:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'ERR_STRING_TOO_LONG'
}
```

**Comment by thlorenz** [comment]
Ah, well in this case it's the actual runtime that limits that.
The only option to fix would be to not load the entire file, but parse it in a streaming manner.
That would require a ton of changes, but I'm open to PRs :)

**Comment by pbadenski** [comment]
Fixed by #20; released in 6.0.0
